stories/core_components/AreaChart.tsx

```diff
@@ -380,4 +380,72 @@ storiesOf('Core components/AreaChart', module)
                 />
             </div>
         )
+    ))
+    .add('optional stacking chart with two attributes, one measure', () => (
```

Comment:
looks like a good candidate for new category 'Optional Stacking chart' 

consider to move all stories under this category :)

Reply:
my purpose is to test both `<AreaChart/>` and `<Visualization config={{type: 'area'}}/>`